lib/main.dart

```diff
@@ -24,21 +27,37 @@ class _HelloAppState extends State<HelloApp> {
     super.initState();
     if (MOBILE_SDK_KEY != '') {
       final LDConfig config = LDConfigBuilder(MOBILE_SDK_KEY).build();
-      final LDUser user = LDUserBuilder('user key').build();
+
+      // Set up the user properties. This user should appear on your LaunchDarkly users dashboard
+      // soon after you run the demo.
+      final LDUser user = LDUserBuilder('example-user-key')
+      .name('Sandy');
+      .build();
       LDClient.start(config, user).whenComplete(() {
           updateFlagEvaluation();
           LDClient.registerFeatureFlagListener(FEATURE_FLAG_KEY, updateFlagEvaluation);
-      });
+      }
+      setState(() { message = 'SDK successfully initialized!'});
```

Comment:
Not sure this is the right place for this--I couldn't tell if the SDK initialization happened during the user identification step, or before.